docs/book/introduction.md

```diff
@@ -16,15 +16,135 @@ description: Welcome to ZenML!
 
 If you answered yes to any of the questions above, ZenML is here to help you:
 
-* **ZenML** is an **open-source MLOps framework** built for data scientists, ML Engineers, and MLOps Developers.
-* It comes equipped with a simple and flexible **Python SDK** that helps you to create **portable and production-ready pipelines**.
-* It also features **cloud-** and **tool-agnostic** **interfaces/abstractions** such as orchestration systems, artifact repositories, and model deployment platforms that cover the entire ML journey.
-* With these abstractions, it remains **extensible** and **open to customization**. That is also backed by numerous **collaborative features** that allow you to work not just with your team but also with the community.
-* Moreover, this creates a separation between the different aspects of ML workflows, allowing users to focus on creating their own respective solutions in an efficient manner.
-* Overall, it reduces the complexity of your development lifecycle, all the way **from an experimentation environment to a production-grade solution**.
+**ZenML** is an extensible, open-source MLOps framework for creating portable, 
+production-ready MLOps pipelines. 
+
+By decoupling infrastructure from code, ZenML gives all ML developers in your 
+organization more freedom and independence in how they approach their work:
+
+![ZenML Overview](.gitbook/assets/intro_zenml_overview.png)
+
+{% tabs %}
+
+{% tab title="ZenML for Data Scientists" %}
```

Comment:
I would make the first tab the MLOps Platform Engineer

Reply:
I changed it but personally I liked the DS tab as default tab better since it better showcases what ZenML does at a glance.

Reply:
Changed it back for that reason

Reply:
> Well our target audience is the MLOPs platform engineer so it should showcase it if it doesnt.

True but ZenML might be used by people from a variety of backgrounds, maybe you're a CTO, or product manager, or a student learning about MLOps, then the platform tab would not be too informative IMO. If you're literally a platform engineer, wouldn't you click on the respective tab right away?

> How about adding a pipeline with one or two basic steps in the MLOps platform engineer tab and showcasing the stack switching as well?

Stack switching is already there under the vendor lock problem, but I don't think it makes sense to add pipelining features here since the platform engineer is not going to write pipelines, so this might not be relevant to them.